skimage/registration/_tvl1.py

```diff
@@ -0,0 +1,206 @@
+# coding: utf-8
+"""TV-L1 optical flow algorithm implementation.
+
+"""
+
+from functools import partial
+import numpy as np
+from scipy import ndimage as ndi
+from skimage.transform import warp
+
+from .utils import coarse_to_fine
+
+
+def _tvl1(I0, I1, flow0, dt, lambda_, tau, nwarp, niter, tol, prefilter):
+    """TV-L1 solver for optical flow estimation.
+
+    Parameters
+    ----------
+    I0 : ~numpy.ndarray
+        The first gray scale image of the sequence.
+    I1 : ~numpy.ndarray
+        The second gray scale image of the sequence.
+    flow0 : ~numpy.ndarray
+        Vector field initialization.
```

Comment:
You can write array shapes here, eg "image1 : array, shape (M, N[, P[, ...]])", then "flow0 : array, shape (D, M, N[, P[, ...]])"

Reply:
Could we set a default for `flow0`? The same for the other variables (except the images). It would be easier to test the function if there's some default values around.

Reply:
@alexdesiqueira, `flow0` is updated  in the coarse to fine schema: at each scale move the flow estimated at small resolution is rescaled to the higher resolution and used as initialization.
Zeros is a reasonable default initial value for  `flow0`  at coarsest level.